integration-tests/byoid/integration_test.go

```diff
@@ -49,10 +47,10 @@ import (
 )
 
 const (
-	envCredentials  = "GCLOUD_TESTS_GOLANG_KEY"
+	envCredentials  = "GOOGLE_APPLICATION_CREDENTIALS"
 	envAudienceOIDC = "GCLOUD_TESTS_GOLANG_AUDIENCE_OIDC"
 	envAudienceAWS  = "GCLOUD_TESTS_GOLANG_AUDIENCE_AWS"
-	envProject      = "GCLOUD_TESTS_GOLANG_PROJECT_ID"
+	envProject      = "GOOGLE_CLOUD_PROJECT"
```

Comment:
At top of TestMain call flag.Parse(). Then you can check testing.Short(). If true just run tests without setup. That way they will just be skipped.